### DISCUSSION TITLE
$apply groupby not working

### DISCUSSION THREAD

**Discussion by kbhaggan** [AutoMapper/AutoMapper.Extensions.OData#164] [Q&A]
Hi

We have an issue with $apply=groupby(). It seems the groupby is not taking place. When use `EnableQuery` it works but without `EnableQuery` it is not working.

Example code:

    [HttpGet]
    [Route("EmsLogs")]
    [ProducesResponseType(typeof(IEnumerable<EmsLog>), StatusCodes.Status200OK)]
    public async Task<IActionResult> Get(ODataQueryOptions<EmsLog> queryOptions, CancellationToken cancellationToken = default)
    {
      queryOptions.Validate(new DefaultValidationSettings());
      var smartAppIds = await _smartAppCacheProvider.GetAsync(cancellationToken);
      var result = await _repository
          .AsQueryable(new EmsLogViewSmartAppIdSpecification(smartAppIds))
          .GetQueryAsync(_mapper, queryOptions, new DefaultQuerySettings());

      return Ok(result);
    }

We see this PR for over a year: 
https://github.com/AutoMapper/AutoMapper.Extensions.OData/pull/108

Is there a workaround for this?

**Comment by BlaiseD** [comment]
None that I know of.  Also see #93 and  #107.  PRs welcome.
